### DISCUSSION TITLE
console corruption issues and solution

### DISCUSSION THREAD

**Discussion by psy0rz** [zbm-dev/zfsbootmenu#644] [General]
### ZFSBootMenu build source

Release EFI

### ZFSBootMenu version

2.3.0

### Boot environment distribution

Alpine

### Problem description

On an older box (HP microserver gen 8) i had some console font corruption issues.

After hours of debugging i found the solution(s):
1. (not a zfsbootmenu issue) I had to blacklist the radeon module, otherwise there would be no output at all at a certain point.
2. Using  boot parameter ```zbm.autosize=0``` made the issue better: This prevents the whole screen from shifting/wrapping up 3 rows.
3. Using  boot parameter ```rd.vconsole.font=none``` solved the characters from being corrupt. (autosize=0 isnt needed anymore at this point)

So i would suggest making ```rd.vconsole.font=none``` a  suggestion as default option, in case people experience these problems. 

I cant believe how much stuff i had to do to get simple console output on this box :)

### Steps to reproduce

1. Insist on using too obsolete hardware
2. Be stubborn.

**Comment by zdykstra** [comment]
I'm not quite clear where you blacklisted `radeon`. That shouldn't be part of any release images specifically because GPUs rarely handle being reinitialized by the kernel in the boot environment.

We do have a wiki 'quirks' page that would be a good place to add an entry for your specific hardware. Right now, there isn't a great place on our docs.zfsboot.menu site for that. It'll take some wrangling to find a place for it.

**Comment by psy0rz** [comment]
The radeon problem had nothing to do with zfsbootmenu but made stuff a lot harder to debug.
